test/db-common.sh

```diff
@@ -8,6 +8,5 @@ function die() {
 
 SERVICES="sa
 policy"
-DBENVS="development
-test
+DBENVS="test
```

Comment:
Right now when you run e.g. start.py it uses the integration DBs, not development ones. Currently nothing uses them, so I deleted them to make create_db run faster.